latest/index.html

```diff
@@ -920,6 +935,59 @@ <h3>Data Extension Responses</h3>
         </p>
       </section>
     </section>
+    <section>
+      <h2>Security Considerations</h2>
+
+      <section>
+        <h3>Preventing Abuse</h3>
+        <ul>
+          <li>Reconciliation services SHOULD guard against too frequent requests to prevent [[DoS]] attacks (see <a href="#error-handling-and-rate-limiting">Error Handling and Rate-limiting</a>).</li>
+          <li>Reconciliation services SHOULD NOT parse the JSON requests sent by clients using a code execution mechanism like JavaScript's <code>eval()</code>, since requests could contain malicious code.</li>
+          <li>Reconciliation clients that display data from <a href="#preview-service">preview services</a> SHOULD ensure that the data is encoded, sanitized, or isolated to prevent [[XSS]] and [[CSRF]] attacks.</li>
+          <li>Reconciliation clients SHOULD provide a configuration option to disallow calls to localhost, with the default set to disallow, as a malicious service could cause the client to make arbitrary POST requests to itself.</li>
+        </ul>
+      </section>
+
+      <section>
+        <h3>Other considerations</h3>
+        <ul>
+          <li>Reconciliation services and clients MAY support user authentication (see <a href="#authentication">authentication</a>).</li>
+          <li>Reconciliation features don't expose information about the underlying platform to origins, they don't allow origins to send data to the underlying platform, access other devices, device sensors, a user agent's native UI,
```

Comment:
By "origin" do you mean "client"? That would read more natural to me.

Reply:
Yeah, it's confusing. In this case, I think it's both client and server for the first two instances and server for the third instance (see [here](https://www.w3.org/TR/security-privacy-questionnaire/) & [here](https://datatracker.ietf.org/doc/html/draft-ietf-websec-origin) for some background). I'll turn this into a draft for now and revisit it later. Maybe we could add an explanation of origin, or I'll reword to use client and server only. Thanks for the review!

Reply:
I removed the confusing 'origin' terminology, added another security consideration that came to mind while rewording (which already shows the benefit of clearer language here!), and linked to some existing definitions for consistency. Would be great if you could take another look!